hr_skill/views/hr_employee.xml

```diff
@@ -0,0 +1,34 @@
+<openerp>
+  <data>
+
+    <record model="ir.ui.view" id="view_employee_skill_form">
+        <field name="name">hr.employee.skill.form</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page string="Skills">
+                    <field name="skill_ids" />
+                </page>
+            </notebook>
+        </field>
+    </record>
+
+    <record id="hr_employee_normal_action_tree" model="ir.actions.act_window">
+        <field name="name">Employees</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">hr.employee</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[('skill_ids','=', active_ids)]</field>
```

Comment:
Is it not better to use something like child_of instead of `=`
or to get this action only on skill of view skill ?